src/material-experimental/mdc-helpers/_focus-indicators.scss

```diff
@@ -80,6 +81,9 @@
   // For options, render the focus indicator when the class .mat-mdc-option-active is present.
   .mat-mdc-focus-indicator.mat-mdc-option-active::before,
 
+  // In the slider focus lands on the thumbs, but we show the indicator on the wrapper element.
```

Comment:
wouldn't it make sense to show it on the thumb that has focus?

Reply:
This is handled in the same way as the non-MDC slider. I think the logic there was that the thumb already has focus indication, but users might have trouble finding the slider itself.

Reply:
The difference is the non-MDC slider only has one thumb. Since the MDC-based one can have 2 I think it makes more sense to put the indicator on the focused thumb. Similar to how we put it on the focused radio button, not the whole radio group

Reply:
I gave it a quick shot and I think there are a couple of problems:
1. It's a little difficult to notice, because it has the same color as the two circles and the bar.
2. It overlaps the tooltip with the value.

![Angular_Material_-_Google_Chrome_2021-06-22_19-35-22](https://user-images.githubusercontent.com/4450522/122973038-16baed80-d391-11eb-857b-9f0b49e06dc4.png)

Reply:
@crisbeto Could you share a screenshot of what the alternative looks like?

Reply:
Here's the same example with the current approach:
![Angular_Material_-_Google_Chrome_2021-06-22_22-34-08](https://user-images.githubusercontent.com/4450522/122995513-006d5b80-d3aa-11eb-9199-cafe39975ace.png)

Reply:
Hmm, I'm not sure. Looks good either way or perhaps even with both ¯\_(ツ)_/¯

I'll defer to @mmalerba or @jelbourn